### PR TITLE
remove department and project tags

### PR DESCRIPTION
**Pull request by zaro0508**
Project and Department tags cannot contain spaces. 
we remove them with an updated template

### CODE REVIEW

File: config/prod/minidream-rstudio-2023-ec2-alb.yaml [hunk 1]
```diff
@@ -8,8 +8,6 @@ stack_name: "minidream-rstudio-2023-ec2-alb"
 dependencies:
   - "prod/minidream-rstudio-2023-ec2.yaml"
 parameters:
-  Department: "SCCE"
```
Comment (xschildw) [inline config/prod/minidream-rstudio-2023-ec2-alb.yaml:11]: You need a new version of the aws-infra script for that.

Comment (zaro0508) [reply config/prod/minidream-rstudio-2023-ec2-alb.yaml:11]: ok, then i'm just gonna remove the spacing in the tags

File: config/prod/minidream-rstudio-2023-ec2-alb.yaml [hunk 2]
```diff
@@ -9,7 +9,7 @@ dependencies:
   - "prod/minidream-rstudio-2023-ec2.yaml"
 parameters:
   Department: "SCCE"
-  Project: "MC2 Center"
+  Project: "MC2Center"
```
Comment (xschildw) [inline config/prod/minidream-rstudio-2023-ec2-alb.yaml:12]: @jesusaurus pushed v7.0 in aws-infra, I'd use that and remove the 2 params.

Comment (zaro0508) [reply config/prod/minidream-rstudio-2023-ec2-alb.yaml]: done

**Pull request merged**